src/data-structures/dto/IImageDTO.ts

```diff
@@ -9,6 +9,7 @@ export enum ImageType {
   VIDEO = 'video',
 }
 export interface IImageDTO {
+  id: string;
```

Comment:
the ID always is a number, may be there is a need to change type to number

Reply:
usually GUIDs are numbers

Reply:
Time-based (Version 1) GUIDs
Random (Version 4) GUIDs - generated using random numbers